### ISSUE TITLE
Question about utility classes and responsive

### ISSUE THREAD

**Issue by botmaster** [kevin-powell/website-starter-template-v3#1]
Hello,

Thank you for sharing this repo. It's really useful and inspiring.
Just one question.
Maybe I am wrong but I dont see any spacing utility classes that are responsive.
Ex: `margin-inline-1` but no `margin-inline-1@small` or `small:margin-inline-1`.

Maybe I m missing something.

**Comment by kevin-powell** [comment]
Correct, I don't have any. It would be possible to add them in if needed, but I'm not a big fan of having media queries within my classes, once I hit that stage of things, I tend to simply have things set up within a component class, or simply have more fluid spacing.
